modules/nextflow/src/main/groovy/nextflow/extension/PublishOp.groovy

```diff
@@ -214,7 +219,7 @@ class PublishOp {
             else {
                 log.warn "Invalid extension '${ext}' for index file '${indexPath}' -- should be CSV, JSON, or YAML"
             }
-            session.notifyFilePublish(indexPath)
+            session.notifyFilePublish(indexPath, null, publishOpts.tags as Map)
```

Comment:
Should this be `annotations` instead of `tags`?

Reply:
Fixed in last commit